src/_guides/language/coming-from/swift-to-dart.md

```diff
@@ -790,14 +786,16 @@ func multiply(_ a: Int, _ b: Int, c: Int = 1, d: Int = 1) -> Int {
 }
 ```
 
-Named parameters must have a default value,
-be marked as nullable (which defaults to a `null` value,
-if not explicitly provided), or be marked as `required`
-for the function to compile.
-Nullable types and [null safety](#null-safety)
-are covered in a later section.
+For the function to compile in Dart, named parameters must include
```

Comment:
The "to compile" part is confusing. Any violations of this are flagged during analysis, not compilation.

I'd just say "Named parameters must include...:"